js/ui/date_box/ui.date_box.base.js

```diff
@@ -779,7 +787,8 @@ var DateBox = DropDownEditor.inherit({
                 this._invalidate();
                 break;
             case "displayFormat":
-                this._updateValue();
+                this.option("text", this._getDisplayedText(this.dateOption("value")));
```

Comment:
Should we replace text option updating to the `renderInputValue` method? I think, the text option is the input value